cmd/waku/flags.go

```diff
@@ -298,6 +298,19 @@ var (
 		Destination: &options.Store.DatabaseURL,
 		EnvVars:     []string{"WAKUNODE2_STORE_MESSAGE_DB_URL"},
 	})
+	StoreMessageDBVacuum = altsrc.NewBoolFlag(&cli.BoolFlag{
+		Name:        "store-message-db-vacuum",
+		Usage:       "Enable database vacuuming at start. Only supported by SQLite database engine.",
```

Comment:
Looks like postgres also supports vacuum https://www.postgresql.org/docs/current/sql-vacuum.html. 
Maybe we should allow this for postgres as well.

Reply:
Yes it could be good to have it also on Postgres.